html/register_account_details.html

```diff
@@ -0,0 +1,139 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+
+    <link
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta2/dist/css/bootstrap.min.css"
+      rel="stylesheet"
+      integrity="sha384-BmbxuPwQa2lc/FVzBcNJ7UAyJxM6wuqIj61tLrc4wSX0szH/Ev+nYRRuWlolflfl"
+      crossorigin="anonymous"
+    />
+    <link rel="stylesheet" href="../css/register_account_details.css" />
+
+    <title>Register</title>
+  </head>
+
+  <body>
+    <div class="register_image"></div>
+    <div
+      aria-live="polite"
+      aria-atomic="true"
+      class="d-flex justify-content-center align-items-center m-2 toasts"
+    >
```

Comment:
please use consistent formating, e.g like this
`<div aria-live="polite" aria-atomic="true" class="d-flex justify-content-center align-items-center m-2 toasts">`